doc/Database.md

```diff
@@ -76,6 +76,23 @@ Stores the pending transaction. Should be made readable.
 | val        | BLOB     | Transaction | NOT NULL    | Binary-serialized |
 | fee        | INTEGER  | Amount      | NOT NULL    | fee rate          |
 
+### `banned` table
+
+Stores the currently banned nodes
+
+| Field name | SQL Type | D type      | Attributes  | Comment           |
+|------------|----------|-------------|-------------|-------------------|
+| url        | TEXT     | string      | PRIMARY KEY | endpoint          |
+| until      | INTEGER  | ulong       | NOT NULL    | unix time in secs |
+
+### `whitelisted` table
+
+Stores the currently whitelisted nodes
+
+| Field name | SQL Type | D type      | Attributes  | Comment           |
+|------------|----------|-------------|-------------|-------------------|
+| url        | TEXT     | string      | PRIMARY KEY | endpoint          |
+
```

Comment:
It could be one table but I think it is cleaner this way.

Reply:
Why is it cleaner ? Having a table with a single entry seems a bit odd to me.
If it was holding more data, I could see an argument for it, but right now it's just using a table as an array.

The more general point though is that instead of having `banned` and `whitelisted` table, we probably should have a `peers` table where we store general information about our peers, of which banned/whitelisted is just a property (could be an enum / bitfield).

That table can later be extended to include data about latency, frequency of connection, known block height, etc...

Reply:
Shall I raise an issue for that. I don't see it as urgent for getting PROD running smoothly. It is now easy to query what nodes we have whitelisted and banned without restarting the node (which is required currently).

Reply:
Let's hold off on this for the time being.

Reply:
Raised https://github.com/bosagora/agora/issues/2642